src/platform_impl/macos/view.rs

```diff
@@ -442,7 +442,9 @@ extern "C" fn unmark_text(this: &Object, _sel: Sel) {
     unsafe {
         let marked_text: id = *this.get_ivar("markedText");
         let mutable_string = marked_text.mutableString();
-        let _: () = msg_send![mutable_string, setString:""];
+        let s: id = msg_send![class!(NSString), new];
+        let _: () = msg_send![mutable_string, setString: s];
+        let _: () = msg_send![s, release];
```

Comment:
You probably know that `let _: () = ` can be written as `let () = `

But I understand that keeping consistency with the old stuff is probably better here.

Reply:
Yeah, this was for consistency, though also because I find `let _: () = ` easier to parse.

Reply:
I really like `let () =`. I feel cheeky when I write it down.